Documentation/helm-operator.md

```diff
@@ -130,6 +130,14 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.rbdGrpcMetricsPort`           | Ceph CSI RBD driver GRPC metrics port.                                                                                      | `9090`                                                 |
 | `csi.rbdLivenessMetricsPort`       | Ceph CSI RBD driver metrics port.                                                                                           | `8080`                                                 |
 | `csi.enableSnapshotter`            | Enable deployment of snapshotter container in ceph-csi provisioner.                                                         | `true`                                                 |
+| `csi.csiRBDProvisionerPluginResource`     | CEPH CSI RBD provisioner plugin resource requirement list.                                                                  | <none>                                                 |
+| `csi.csiRBDProvisionerSidecarResource`    | CEPH CSI RBD provisioner sidecar resource requirement list.                                                                 | <none>                                                 |
```

Comment:
I'm still concerned about specifying a single resource spec for all the sidecars in a driver. As complicated as it makes things, if we're implementing the resources, seems like we need to allow full flexibility. If you apply the same resource requests/limits to all the sidecars, you would have to overprovision the resources by several times just for the sake of one sidecar that needs more resources.

Consider the containers for the rbd driver:
- The rbd provisioner has five containers: csi-rbdplugin, csi-provisioner, csi-resizer, csi-rbdplugin-attacher, csi-snapshotter, and liveness-prometheus
   - Does it make sense for the provisioner, resizer, attacher, snapshotter, and liveness containers to have the same resource limits? They could all have very different requirements. The liveness probe is certainly very low. The resizer and snapshotter would only depend on usage. The attacher would also depend on usage. 
- The rbd plugin has three containers: csi-rbdplugin, driver-registrar, and liveness-prometheus
   - The registrar and liveness containers both seem like they should have very low requirements. Could we set the example limits much lower for them? For example, what about 50MB memory and 1/10 CPU for the rbd plugin resource examples?

The cephfs provisioner and plugins have similar containers so the same would apply to them.